### PR TITLE
Smartyads Bid Adapter: add coppa field from config

### PR DESCRIPTION
**Pull request by SmartyAdsSSP**
## Type of change
- [ x] Feature

## Description of change

Add coppa field from config

- test parameters for validating bids
```
var adUnits = [
                // Will return static native ad. Assets are stored through user UI for each placement separetly
                {
                    code: 'placementId_0',
                    mediaTypes: {
                        native: {}
                    },
                    bids: [
                        {
                            bidder: 'smartyads',
                            params: {
                                placementId: 0,
                                traffic: 'native'
                            }
                        }
                    ]
                },
                // Will return static test banner
                {
                    code: 'placementId_0',
                    mediaTypes: {
                        video: {
                            sizes: [[300, 250]],
                        }
                    },
                    bids: [
                        {
                            bidder: 'smartyads',
                            params: {
                                placementId: 0,
                                traffic: 'video'
                            }
                        }
                    ]
                },
                // Will return test vast xml. All video params are stored under placement in publishers UI
                {
                    code: 'placementId_0',
                    mediaTypes: {
                        banner: {
                            sizes: [[300, 250]],
                        }
                    },
                    bids: [
                        {
                            bidder: 'smartyads',
                            params: {
                                placementId: 0,
                                traffic: 'banner'
                            }
                        }
                    ]
                }
            ];
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- support@smartyads.com
- [x ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: test/spec/modules/smartyadsBidAdapter_spec.js [hunk 1]
```diff
@@ -57,6 +59,19 @@ describe('SmartyadsAdapter', function () {
       expect(data.placements).to.be.an('array').that.is.empty;
     });
   });
+
+  describe('with COPPA', function() {
+    it('should send the Coppa "required" flag set to "1" in the request', function () {
+      sinon.stub(config, 'getConfig')
+        .withArgs('coppa')
+        .returns(true);
```
Comment (robertrmartinez) [inline test/spec/modules/smartyadsBidAdapter_spec.js:67]: It is a better idea to set your stubs in beforeEach and reset them in afterEach.

If for some reason your test fails before the `config.getConfig.restore();` then the next adapter tests which run will have this stubbed still which can cause flakey failures.

**Review by robertrmartinez — CHANGES_REQUESTED** [review]
Other than the minor test update looks good.

**Comment by SmartyAdsSSP** [comment]
Hello, @robertrmartinez ! Did you have chance to review my fixes?

**Review by robertrmartinez — APPROVED** [review]
thanks for the update!

Approved, waiting second review

**Pull request merged**